apps/test-suite/tests/FileSystemNext.ts

```diff
@@ -0,0 +1,95 @@
+'use strict';
+import * as FS from 'expo-file-system';
+import { File, Directory } from 'expo-file-system/next';
+import { Platform } from 'react-native';
+
+export const name = 'FileSystem@next';
```

Comment:
Funny enough, this name cannot contain spaces for some weird reason (it is picked up as 2 separate test cases), so I named it like this. Assuming it's OK but can investigate if not.